test/release-testdata.json

```diff
@@ -0,0 +1,118 @@
+{
+  "ItemType": "Release",
+  "IsStale": false,
+  "TotalResults": 2,
+  "ItemsPerPage": 30,
+  "Items": [
+    {
+      "Id": "Releases-2",
+      "Assembled": "2016-10-12T23:05:36.565+00:00",
+      "ReleaseNotes": null,
+      "ProjectId": "Projects-200",
+      "ChannelId": "Channels-823",
+      "ProjectVariableSetSnapshotId": "variablesetstuff",
+      "LibraryVariableSetSnapshotIds": [
+        "variableset-LibraryVariableSets-102-s-10-PF6RH",

```

Comment:
Ditto, can you trim some of these out so its a smaller file / just seems like a bunch of waste